hw/ip/i2c/dv/README.md

```diff
@@ -65,27 +61,27 @@ The `i2c_base_vseq` virtual sequence is extended from `cip_base_vseq` and serves
 All test sequences are extended from `i2c_base_vseq`.
 It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
 Some of the most commonly used tasks / functions are as follows:
-* task 1:
-* task 2:
+* `i2c_init`: Setup the control register, identifier and clear the FIFOs.
+* `wait_{host|target}_for_idle`.
+
+There's also another test sequence which multiple other tests use as a base and that is `i2c_rx_tx_vseq`.
+This sequence is useful for tests that want to generate receive or transmit packets.
 
 #### Functional coverage
 To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
-The following covergroups have been developed to prove that the test intent has been adequately met:
-* cg1:
-* cg2:
+Please see the [testplan](../data/i2c_testplan.hjson) for details of the covergroups.
 
 ### Self-checking strategy
 #### Scoreboard
 The `i2c_scoreboard` is primarily used for end to end checking.
-It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
-* analysis port1:
-* analysis port2:
+It has a TLUL interface to monitor interactions with the registers.
```

Comment:
I mostly see this way across the documentation.
```suggestion
It has a TL-UL interface to monitor interactions with the registers.
```

Reply:
Thanks, changed.